src/babel.ts

```diff
@@ -13,12 +13,17 @@ export default function transform (opts: TransformOptions): TRANSFORM_RESULT {
       [require('@babel/plugin-transform-modules-commonjs'), { allowTopLevelThis: true }],
       [require('babel-plugin-dynamic-import-node'), { noInterop: true }],
       [require('babel-plugin-transform-import-meta')],
-      [require('@babel/plugin-syntax-class-properties')]
+      [require('@babel/plugin-proposal-class-properties'), { loose: true }]
```

Comment:
Is this change necessary for decorators' support? (if yes, please provide an example) Since modern environments already support class properties, we only add syntax support.

Reply:
I was following Babel docs which recommended `proposal` over `syntax`, but these may be out of date.  Verified that `syntax` appears to be compatible with the decorators proposal, so I've switched it back.